tests/SprykerEcoTest/Zed/Adyen/Business/Facade/AdyenFacadeExecutePostSaveHookTest.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEcoTest\Zed\Adyen\Business\Facade;
+
+use Generated\Shared\DataBuilder\AdyenApiResponseBuilder;
+use Generated\Shared\Transfer\AdyenApiMakePaymentResponseTransfer;
+use Generated\Shared\Transfer\AdyenApiResponseTransfer;
+use SprykerEco\Zed\Adyen\Dependency\Facade\AdyenToAdyenApiFacadeBridge;
+use SprykerEcoTest\Zed\Adyen\Business\BaseSetUpTest;
+
+/**
+ * @group Functional
+ * @group SprykerEco
+ * @group Zed
+ * @group Adyen
+ * @group Business
+ */
+class AdyenFacadeExecutePostSaveHookTest extends BaseSetUpTest
+{
+    /**
+     * @var string
+     */
+    protected const PROCESS_NAME_ADYEN_CREDIT_CARD = 'AdyenPayPal01';
+
+    /**
+     * @var string
+     */
+    protected const OMS_STATUS_NEW = 'new';
+
+    /**
+     * @var string
+     */
+    protected const ADYEN_PAYMENT_STATUS_AUTHORISED = 'Authorised';
+
+    /**
+     * @var string
+     */
+    protected const ADYEN_PAYMENT_STATUS_REFUSED = 'Refused';
```

Comment:
It looks like the unused constant. Please remove it.

Reply:
Added usege.